packages/babel-cli/test/exit-loader.cjs

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line no-process-exit
```

Comment:
```suggestion
// We need to gracefully exit rather than just terminating spawn tests with SIGINT
// to let c8 correctly report the collected coverage info.
// eslint-disable-next-line no-process-exit
```